secretmanager/api/SecretManagerSample/Program.cs

```diff
@@ -362,6 +380,88 @@ public static void GetSecret(string projectId, string secretId)
         }
         // [END secretmanager_get_secret]
 
+        // [START secretmanager_iam_grant_access]
+        /// <summary>
+        /// Grant a user or account access to the secret.
+        /// </summary>
+        /// <param name="projectId">ID of the project where the secret resides.</param>
+        /// <param name="secretId">ID of the secret.</param>
+        /// <param name="member">IAM member to grant with user: or serviceAccount: prefix</param>
+        /// <example>
+        /// Grant a user or account access to the secret.
+        /// <code>IAMGrantAccess("my-project", "my-secret", "user:foo@example.com")</code>
+        /// </example>
+        public static void IAMGrantAccess(string projectId, string secretId, string member)
+        {
+            SecretManagerServiceClient client = SecretManagerServiceClient.Create();
+
+            // Create the request to get the current IAM policy.
+            var getRequest = new GetIamPolicyRequest
+            {
+                ResourceAsResourceName = new SecretName(projectId, secretId),
```

Comment:
This should probably be called something like SecretAsSecretName, so there's probably something missing in the proto annotations. Nothing to do with you though. Let me check internally.

Reply:
Nope, scratch that, this is alright since the GetIamPolicyRequest message is from the IAM API and it's used to get policies for all kinds of resources.